tools/tests/test_lint.py

```
@@ -127,6 +127,23 @@ def test_relative_reference_within_own_skill_is_clean(tmp_path):
+    other = tmp_path / "skills" / "other-skill"
+    other.mkdir(parents=True)
+    (other / "SKILL.md").write_text(
+        "See https://github.com/anthropics/skills/tree/main/skills/pdf/SKILL.md\n"
+        "The upstream-skills/bar/ layout and their-repo/skills/baz/ are fine.\n",
+        encoding="utf-8",
+    )
+    assert lint.run(tmp_path) == []
+    # ...and a true sideways reference still fires.
+    (other / "SKILL.md").write_text("Load skills/example-skill/ first.\n", encoding="utf-8")
+    findings = lint.run(tmp_path)
+    assert len(findings) == 1 and "sideways-dep" in findings[0]
```

Ruff (0.16.1)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)